### DISCUSSION TITLE
Update AdGuard Docker

### DISCUSSION THREAD

**Discussion by MrNobody334** [AdguardTeam/AdGuardHome#5201] [General]
Hi. 
Do we need update AdGuardHome Docker container manually? if so, how can we update? 
Just pulling the container?

**Comment by 26adriano** [comment]
if you check this out --> https://github.com/AdguardTeam/AdGuardHome/wiki/Docker#update.
They say to do it manually. I guess you could use watchtower to automate the process, but will need to test that.

**Comment by bcookatpcsd** [comment]
```
time="2022-12-15T18:25:36-05:00" level=info msg="Found new adguard/adguardhome:edge image (b56250830346)"
time="2022-12-15T18:25:43-05:00" level=info msg="Stopping /adguard (4746b403f2be) with SIGTERM"
time="2022-12-15T18:25:44-05:00" level=info msg="Creating /adguard"
```

No problem that I've run into..
